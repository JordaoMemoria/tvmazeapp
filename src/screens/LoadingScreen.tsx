import React from 'react';
import styled from 'styled-components/native';
import Loading from '../components/Loading';

export default function LoadingScreen() {
  return (
    <View>
      <Loading margin={0} />
    </View>
  );
}

const View = styled.View`
  justify-content: center;
  align-items: center;
  flex: 1;
`;
