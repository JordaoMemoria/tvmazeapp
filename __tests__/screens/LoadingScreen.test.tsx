import React from 'react'
import LoadingScreen from '../../src/screens/LoadingScreen'
import renderer from 'react-test-renderer'

jest.useFakeTimers()

describe('<LoadingScreen />', () => {
  it('Renders correctly', () => {
    const tree = renderer.create(<LoadingScreen />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
