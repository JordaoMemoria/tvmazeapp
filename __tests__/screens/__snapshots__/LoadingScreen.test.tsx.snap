// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingScreen /> Renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <ActivityIndicator
    color="#5AB5E8"
    margin={0}
    size="large"
    style={
      Array [
        Object {
          "marginTop": 0,
        },
      ]
    }
  />
</View>
`;
